WORKSPACE

```diff
@@ -280,6 +280,7 @@ haskell_register_ghc_nixpkgs(
     ],
     repositories = dev_env_nix_repos,
     version = "8.6.5",
+    is_static = True,
```

Comment:
This is no longer auto-detected. See https://github.com/tweag/rules_haskell/pull/970#pullrequestreview-259387269.